packages/zoe/test/swingsetTests/zoe/vat-dave.js

```diff
@@ -87,13 +87,16 @@ const build = async (E, log, zoe, issuers, payments, installations, timer) => {
       const installationCode = await E(zoe).getInstallation(installationHandle);
       // pick some arbitrary code points as a signature.
       assert(
-        installationCode.indexOf("['Asset', 'Price']") === 58060,
-        details`source code didn't match`,
+        installationCode.indexOf("['Asset', 'Price']") > 5000,
```

Comment:
Could we do something like:

```suggestion
        installationCode.includes("['Asset', 'Price']")
```
And not try to pinpoint the line number at all right now?

Reply:
done